flask_opentracing/tracing.py

```diff
@@ -19,6 +19,7 @@ def __init__(
         app=None,
         traced_attributes=[],
         start_span_cb=None,
+        trace_response_header=False,
```

Comment:
Do we want to have it disabled by default?
I am supposed to enable it by default in https://github.com/signalfx/splunk-otel-go per https://github.com/signalfx/splunk-otel-go/pull/18#issuecomment-808338996

Reply:
Java docs say it is disabled by default. I copied that.

Reply:
Enabling by default makes sense to me though. Will do that and confirm internally.

Reply:
OK. Keep me in the loop as if it should not be enabled by default then https://github.com/signalfx/splunk-otel-go/pull/18#issuecomment-808338996 does not probably make sense.